pallets/offchain-task-scheduler/src/mock/utils.rs

```diff
@@ -1,23 +1,21 @@
 use super::runtime::{AccountId, Runtime};
 use crate::GenesisConfig;
+use frame_support::traits::GenesisBuild;
 use runtime_utils::ExtBuilder;
 use sp_keystore::SyncCryptoStore;
 use sp_runtime::{traits::IdentifyAccount, RuntimeAppPublic};
 
-pub(crate) fn generate_authority(
-	builder: &mut ExtBuilder<GenesisConfig<Runtime>>,
-) -> sp_core::sr25519::Public {
-	const PHRASE: &str =
-		"news slush supreme milk chapter athlete soap sausage put clutch what kitten";
+pub(crate) fn generate_authority(builder: &mut ExtBuilder, n: u32) -> sp_core::sr25519::Public {
```

Comment:
Q: Is `n` supposed to be the seed for a given key ?

Reply:
yes